django2pydantic/handlers/property.py

```
@@ -1,31 +1,34 @@
 
 
-class PropertyHandler(FieldTypeHandler[property]):
+class PropertyHandler(FieldTypeHandler[type[property]]):
     """Handler for property decorated methods."""
 
-    @override
     @classmethod
-    def field(cls) -> type[property]:
-        return property
+    @override
+    def field(cls):
```

Ruff (0.8.2)

Missing return type annotation for classmethod `field`

(ANN206)